pyproject.toml

```diff
@@ -7,7 +7,6 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-irc = "^20.1.0"
```

Comment:
Wasn't this a dependency? What did we use this for?

Reply:
For irc client that we removed